config/production.yaml

```diff
@@ -20,3 +20,5 @@ subLoggerLevel.http: info
 subLoggerLevel.jaeger: info
 subLoggerLevel.tchannel: info
 useDatacenter: false
+#Some tchannel clients have retryCount functionality, this feature will be disabled by default
```

Comment:
1) `have retry functionality` or `support retry` instead of `have retryCount functionality`

2) You can just say `tchannel client retry is disabled by default`